app/src/main/java/org/jellyfin/mobile/utils/AndroidVersion.kt

```diff
@@ -0,0 +1,32 @@
+package org.jellyfin.mobile.utils
+
+import android.os.Build
+
+object AndroidVersion {
```

Comment:
We should probably add a kdoc to each of the vals to specify both the name & API level.